bindings/python/pymongoarrow/monkey.py

```diff
@@ -11,12 +11,30 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Add PyMongoArrow APIs to PyMongo."""
+
 __all__ = [
     'patch_all'
 ]
 
 
 def patch_all():
+    """Patch all PyMongoArrow methods into PyMongo.
+
+    This function should be called **before** ``pymongo`` is imported for
+    the first time. Failing to do so might lead to unexpected results.
```

Comment:
Is this true? What unexpected results? AFAIK we can patch methods onto classes at anytime.

Reply:
Removed.